src/node/desktop/src/assets/locales/fr.json

```diff
@@ -1,113 +1,113 @@
-    "copyImage": "FR [Copy Image]",
-    "reload": "FR [Reload]",
-    "copyImageAddress": "FR [Copy Image Address]",
-    "cut": "FR [Cut]",
-    "copy": "FR [Copy]",
-    "paste": "FR [Paste]",
-    "selectAll": "FR [Select All]",
-    "inspectElement": "FR [Inspect Element]",
-    "errorDownloadingImage": "FR [Error downloading image]",
-    "downloadCancelledMessage": "FR [The download was cancelled.]",
-    "downloadInterruptedMessage": "FR [The download was interrupted.]",
-    "save": "FR [Save]"
+    "saveImageAsDots": "Enregistrer l'image sous...",
+    "saveImageAs": "Enregistrer l'image sous",
+    "copyImage": "Copier l'image",
+    "reload": "Recharger",
+    "copyImageAddress": "Adresse de l'image à copier",
+    "cut": "Couper",
+    "copy": "Copier",
+    "paste": "Coller",
+    "selectAll": "Tout sélectionner",
+    "inspectElement": "Inspecter l'élément",
+    "errorDownloadingImage": "Erreur de téléchargement de l'image",
+    "downloadCancelledMessage": "Le téléchargement a été annulé",
+    "downloadInterruptedMessage": "Le téléchargement a été interrompu.",
+    "save": "Sauvegarder"
   },
   "chooseRDialog": {
-    "documentTitle": "FR [Choose R Installation]",
-    "title": "FR [RStudio requires an existing installation of R.]",
-    "subtitle": "FR [Please select the version of R to use.]",
-    "use64BitR": "FR [Use your machine's default 64-bit version of R]",
-    "use32BitR": "FR [Use your machine's default 32-bit version of R]",
-    "useCustom": "FR [Choose a specific version of R:]",
-    "customizeRenderingEngine": "FR [You can also customize the rendering engine used by RStudio.]",
-    "renderingEngineLabel": "FR [Rendering Engine:]",
-    "renderingEngineOptionAuto": "FR [Auto-detect (recommended)]",
-    "renderingEngineOptionDesktop": "FR [Desktop]",
-    "renderingEngineOptionSoftware": "FR [Software]",
-    "renderingEngineChangedTitle": "FR [Rendering Engine Changed]",
-    "renderingEngineChangedMessage": "FR [The rendering engine has been changed.\nPlease restart RStudio for these changes to take effect.]",
-    "rLaunchFailedTitle": "FR [Error Loading R]",
-    "rLaunchFailedMessage": "FR [An error occurred while attempting to load the selected version of R. Please select a different R installation.]"
+    "documentTitle": "Choisir l'installation R",
+    "title": "RStudio requiert une installation existante de R.",
+    "subtitle": "Veuillez sélectionner la version de R à utiliser.",
+    "use64BitR": "Utilisez la version 64 bits par défaut de R sur votre machine",
+    "use32BitR": "Utilisez la version 32 bits par défaut de R sur votre machine",
+    "useCustom": "Choisissez une version spécifique de R :",
+    "customizeRenderingEngine": "Vous pouvez également personnaliser le moteur de rendu utilisé par RStudio",
+    "renderingEngineLabel": "Moteur de rendu :",
+    "renderingEngineOptionAuto": "Détection automatique (recommandée)",
+    "renderingEngineOptionDesktop": "Espace de travail",
```

Comment:
Not sure what english version  "Desktop" really means but using rendering engine "Espace de travail" is not that clear to. 
@romainfrancois you did not flag that so maybe I am just missing some context.